lib/elixir/lib/code/formatter.ex

```diff
@@ -1318,7 +1320,12 @@ defmodule Code.Formatter do
       entries =
         case state.sigils do
           %{^name => callback} ->
-            case callback.(hd(entries), sigil: List.to_atom([name]), modifiers: modifiers) do
+            case callback.(hd(entries),
+                   file: state[:file],
+                   line: meta[:line],
```

Comment:
The plugin receives the meta information. They can extract the line number from it manually. :)

Reply:
Actually, I am wrong, please ignore me.